src/windows/functions.cpp

```diff
@@ -393,6 +393,37 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterEjectable(TCHAR driveLetter) {
+  TCHAR devicePath[8];
+  wsprintf(devicePath, TEXT("%c:\\"), driveLetter);
+
+  MountUtilsLog("Checking whether drive is ejectable: " + std::string(1, driveLetter));
+
+  switch (GetDriveType(devicePath)) {
```

Comment:
Yeah, there is a lot of stuff we can re-use, which is the reason I wanted to bring all of that stuff together in a single module that provides "disk" utilities.